### ISSUE TITLE
[1.10.2][Server] Crashes on startup 

### ISSUE THREAD

**Issue by DeadSix27** [InfinityRaider/AgriCraft#715]
Tested newest alpha, server just crashes.

Forge version: `forge-1.10.2-12.18.3.2185`
Agricraft version: `agricraft-2.0.0a1.jar`
Infinitylib version: `infinitylib-0.3.0.jar`
List of mods: _`(Ask me if you need that)`_

Stacktrace:
```
[21:04:02] [Server thread/ERROR] [FML]: Caught exception from AgriCraft (agricraft)
java.lang.NoSuchMethodError: net.minecraft.item.ItemBlock.func_77640_w()Lnet/minecraft/creativetab/CreativeTabs;
        at com.infinityraider.agricraft.utility.CustomWoodType.lambda$init$4(CustomWoodType.java:161) ~[CustomWoodType.class:?]
        at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:267) ~[?:1.8.0_111]
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175) ~[?:1.8.0_111]
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) ~[?:1.8.0_111]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) ~[?:1.8.0_111]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_111]
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151) ~[?:1.8.0_111]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174) ~[?:1.8.0_111]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_111]
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418) ~[?:1.8.0_111]
        at com.infinityraider.agricraft.utility.CustomWoodType.init(CustomWoodType.java:167) ~[CustomWoodType.class:?]
        at com.infinityraider.agricraft.utility.CustomWoodType.init(CustomWoodType.java:139) ~[CustomWoodType.class:?]
        at com.infinityraider.agricraft.proxy.IProxy.initCustomWoodTypes(IProxy.java:53) ~[IProxy.class:?]
        at com.infinityraider.agricraft.proxy.IProxy.initStart(IProxy.java:38) ~[IProxy.class:?]
        at com.infinityraider.infinitylib.InfinityMod.init(InfinityMod.java:100) ~[InfinityMod.class:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
        at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:600) ~[forge-1.10.2-12.18.3.2185-universal.jar:?]
        at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
        at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.10.2.jar:?]
        at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243) ~[forge-1.10.2-12.18.3.2185-universal.jar:?]
        at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221) ~[forge-1.10.2-12.18.3.2185-universal.jar:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
        at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.10.2.jar:?]
        at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145) [LoadController.class:?]
        at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:795) [Loader.class:?]
        at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:107) [FMLServerHandler.class:?]
        at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:333) [FMLCommonHandler.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:214) [ld.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
[21:04:02] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from AgriCraft (agricraft)
Caused by: java.lang.NoSuchMethodError: net.minecraft.item.ItemBlock.func_77640_w()Lnet/minecraft/creativetab/CreativeTabs;
        at com.infinityraider.agricraft.utility.CustomWoodType.lambda$init$4(CustomWoodType.java:161) ~[CustomWoodType.class:?]
        at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:267) ~[?:1.8.0_111]
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175) ~[?:1.8.0_111]
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) ~[?:1.8.0_111]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) ~[?:1.8.0_111]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_111]
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151) ~[?:1.8.0_111]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174) ~[?:1.8.0_111]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_111]
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418) ~[?:1.8.0_111]
        at com.infinityraider.agricraft.utility.CustomWoodType.init(CustomWoodType.java:167) ~[CustomWoodType.class:?]
        at com.infinityraider.agricraft.utility.CustomWoodType.init(CustomWoodType.java:139) ~[CustomWoodType.class:?]
        at com.infinityraider.agricraft.proxy.IProxy.initCustomWoodTypes(IProxy.java:53) ~[IProxy.class:?]
        at com.infinityraider.agricraft.proxy.IProxy.initStart(IProxy.java:38) ~[IProxy.class:?]
        at com.infinityraider.infinitylib.InfinityMod.init(InfinityMod.java:100) ~[InfinityMod.class:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
        at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:600) ~[forge-1.10.2-12.18.3.2185-universal.jar:?]
        at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
        at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.10.2.jar:?]
        at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243) ~[LoadController.class:?]
        at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221) ~[LoadController.class:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_111]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_111]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
        at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[minecraft_server.1.10.2.jar:?]
        at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[minecraft_server.1.10.2.jar:?]
        at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145) ~[LoadController.class:?]
        at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:795) ~[Loader.class:?]
        at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:107) ~[FMLServerHandler.class:?]
        at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:333) ~[FMLCommonHandler.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:214) ~[ld.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
```

**Comment by InfinityRaider** [comment]
So it begins...

Forgot to test server before pushing the new code, thanks for reporting

**Comment by DeadSix27** [comment]
Heh, ye I missed agricraft so when I saw it I immediately tested it, can't wait to see if It works with E-IO etc, if not It's fine, saw the issue stating it will come later (probably) :P

**Comment by InfinityRaider** [comment]
for now there's no compat yet, its only the mod by itself,still alpha, we'll be working on bugfixing and adding compat again now.

**Comment by HiddenCobalt** [comment]
Can confirm. Running over a hundred mods on Forge-1.10.2-12.18.3.2185 but I guess this is not a cross compat issue at the moment. 

[crash-2016-12-10_20.34.40-server.txt](https://github.com/InfinityRaider/AgriCraft/files/644083/crash-2016-12-10_20.34.40-server.txt)

**Comment by DeadSix27** [comment]
@HiddenCobalt 
He fixed it in 2.0.0a2:
- FIXED: Server initialization crash!

**Comment by RlonRyan** [comment]
Well, I don't think we need this anymore!
